elasticapm/traces.py

```diff
@@ -92,7 +96,7 @@ def begin_span(self, name, span_type, context=None, leaf=False):
             return None
 
         start = _time_func() - self.start_time
-        span = Span(self._span_counter - 1, name, span_type, start, context)
+        span = Span(self._span_counter - 1, self.id, self.trace_id, name, span_type, start, context)
```

Comment:
(probably doesn't need saying, but I'll say it just in case)
span ID needs to change to a hex-encoded random 64-bit ID